utils/hare-fetch-fids

```diff
@@ -178,6 +178,15 @@ def parse_opts(argv):
                    'Default: Returns the fids for all the services.',
                    type=str,
                    action='store')
+    p.add_argument('--index',
+                   '-i',
+                   help='service index. - Returns fid for given service'
```

Comment:
I think we need to specifically mention that a simplistic indexing method is used where the index value is used as a subscript into the list of identifiers for a given service type. Hare does not guarantee that a mapping between a given index and service identifier will always be preserved.

Reply:
Done